apps/ledger-live-desktop/tests/models/DiscoverPage.ts

```diff
@@ -145,4 +146,17 @@ export class DiscoverPage {
 
     return result;
   }
+
+  send(request: Record<string, unknown>) {
+    const code = `
+      (function() {
+        return window.ledger.e2e.walletApi.send('${JSON.stringify(request)}');
+      })()
+    `;
+
+    return this.page.evaluate(code => {
```

Comment:
good but you use twice the name `code` (as argument of the first function passed) and passed as the second argument of `evaluate`. Please find another name as it might be confusing